.github/workflows/docs.yml

```
@@ -24,7 +24,7 @@ jobs:
           cd -
 
       - name: Deploy
-        if: github.ref == 'refs/heads/feature/catchem_begins'
+        #if: github.ref == 'refs/heads/feature/catchem_begins'
```

```suggestion
        if: github.ref == 'refs/heads/feature/catchem_begins'
```
I think we may be able put together a system for deploying open PRs separately from the main deployment. But it may be easier to wait until we go to the upstream repo and then we can use RTD to do that.